### PR TITLE
Update apps-advanced-threat-protection-macos.md

### PR DESCRIPTION
**Pull request by msbemba**
Added information about Intune-based deployment for Microsoft Defender for Endpoint on macOS to the next steps

### CODE REVIEW

**Comment by PRMerger20** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by yogkumgit** [comment]
@JohanFreelancer9 copy review

File: memdocs/intune/apps/apps-advanced-threat-protection-macos.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ The app you've created is displayed in the apps list, where you can assign it to
 > Currently, Apple does not provide a way for Intune to uninstall Microsoft Defender for Endpoint on macOS devices.
 
 ## Next steps
+- To learn about Intune-based deployment for Microsoft Defender for Endpoint on macOS, see [Intune-based deployment for MDE on macOS](/microsoft-365/security/defender-endpoint/mac-install-with-intune)
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/apps-advanced-threat-protection-macos.md:85]:
```suggestion
- To learn about Intune-based deployment for Microsoft Defender for Endpoint on macOS, see [Intune-based deployment for Microsoft Defender for Endpoint on macOS](/microsoft-365/security/defender-endpoint/mac-install-with-intune)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger12** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please review and merge this PR. Thanks

**Comment by yogkumgit** [comment]
@Erikre Gentle reminder. Thanks

**Pull request merged**